packages/yew/src/virtual_dom/vtext.rs

```diff
@@ -28,6 +28,20 @@ impl VText {
     }
 }
 
+impl std::fmt::Debug for VText {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        write!(
+            f,
+            "VText {{ text: \"{}\", reference: {} }}",
+            self.text,
+            match &self.reference {
+                Some(_) => "Some(...)",
```

Comment:
Nit: maybe also print the `VText`'s content? Or is that not possible?

Reply:
This was intentional, as it made the debug log output much more unreadable. Especially so in already verbose log output, like the one I added to the layout tests.

Reply:
This was intentional, as it made the debug log output much more unreadable. Especially so in already verbose log output, like the one I added to the layout tests.